requirements/requirements.in

```diff
@@ -38,7 +39,7 @@ python-radius==1.0
 python3-saml==1.4.0
 social-auth-core==3.0.0
 social-auth-app-django==2.1.0
-requests<2.16  # Older versions rely on certify
```

Comment:
@AlanCoding do you know why we pinned this?

https://github.com/ansible/awx/commit/f4a241aba238b2f64e5734f297325098d8e9eda8
https://github.com/ansible/awx/pull/1415

Reply:
https://github.com/ansible/awx/pull/1198#issuecomment-365140621

Is pretty much all there is on background on that.

At the time, this was still an upgrade, because we had been at a lower version at like 2.15. So when I did it, I upgraded it just shy of 2.16. We certainly don't want to stay there. Obviously certify is added here. I would hope that any problems that may or may not have existed then won't still exist.

Reply:
Looks like `azure-keyvault` wants a newer version, so we'll have to see what falls out of this.

Reply:
prolly some nonsense with certify and ceritifcate verification for LDAP